### PR TITLE
CI: Add GitHub Actions config

### PR DESCRIPTION
**Pull request by Turbo87**
This PR replaces TravisCI with GitHub Actions, roughly based upon https://github.com/ember-cli/ember-addon-output/blob/master/.github/workflows/ci.yml

All of the ember-try scenarios were currently failing due to a change in `moment`, so this PR also updates the transitive `moment` dependency. Three of the scenarios are still disabled until we can fix them correctly.

Example run: https://github.com/Turbo87/ember-cp-validations/actions/runs/1416358445

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10.x
```
Comment (dbendaou) [inline .github/workflows/ci.yml:18]: 10 is pretty old and not maintained anymore, maybe we could switch to 12 or 14? 
https://nodejs.org/en/about/releases/

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    name: ${{ matrix.try-scenario }}
+    runs-on: ubuntu-latest
+    needs: ['lint', 'test']
+
+    strategy:
+      fail-fast: false
+      matrix:
+        try-scenario:
+          - ember-lts-2.16
+          - ember-lts-2.18
```
Comment (dbendaou) [inline .github/workflows/ci.yml:48]: Same I think we can drop 2.16 and 2.18 WDYT?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Turbo87) [reply .github/workflows/ci.yml:18]: the TravisCI config was using 10 and I tried to port it as close as possible

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Turbo87) [reply .github/workflows/ci.yml:48]: see above. one thing at a time.

**Review by dbendaou — APPROVED** [review]

**Pull request merged**